### PR TITLE
Add further hiding / closing of apps when screen sharing 

### PR DESCRIPTION
**Pull request by stevysmith**
- Close Gmail Tabs - _closes gmail tabs from open Chrome windows_
- Hide Discord - _hides discord mac app_
- Hide Slack - _hides slack mac app_
- Hide Notes - _hides notes mac app_

### CODE REVIEW

**Comment by mcmillan** [comment]
Hi @stevysmith! Would you mind splitting this up into two PRs – keeping this one for `close-gmail-tabs` / `hide-discord` / `hide-notes` and `hide-slack`, and creating a new one for `slack-share-update`? The first four are pretty straightforward, but the last one might need a bit more reviewing 😄

**Comment by stevysmith** [comment]
Sure thing

File: triggers/close-gmail-tabs/screen-share-started [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+if [ -n "$TUPLE_TRIGGER_IS_SELF" ]; then
+  osascript -l JavaScript <<'END'
+    var chrome = Application('Google Chrome');
+    if (chrome.running()) {
+        var windows = chrome.windows();
+        for (var i = 0; i < windows.length; i++) {
+            var tabs = windows[i].tabs();
+            for (var j = tabs.length - 1; j >= 0; j--) {
+                var tab = tabs[j];
+                if (tab.url().includes("mail.google.com")) {
```
Comment (mcmillan) [inline triggers/close-gmail-tabs/screen-share-started:12]: This has the potential to cause some false positives if for whatever reason another URL happened to contain `mail.google.com`. Could we change this to something a touch more specific, e.g.

```suggestion
                if (tab.url().startsWith("https://mail.google.com/")) {
```

Once that's updated this should be good to go!

**Review by mcmillan — CHANGES_REQUESTED** [review]

**Review by mcmillan — APPROVED** [review]
Thanks very much @stevysmith!

**Pull request merged**